pallets/avn-anchor/src/mock.rs

```diff
@@ -31,9 +89,29 @@ frame_support::construct_runtime!(
         Avn: pallet_avn::{Pallet, Storage, Event},
         AvnProxy: avn_proxy::{Pallet, Call, Storage, Event<T>},
         AvnAnchor: avn_anchor::{Pallet, Call, Storage, Event<T>},
+        TokenManager: pallet_token_manager::{Pallet, Call, Storage, Event<T>},
```

Comment:
it must have been fun to implement all these :smile:

Reply:
:D